draft-packages/tag/KaizenDraft/Tag/Tag.scss

```diff
@@ -73,8 +73,8 @@ $small: $ca-grid;
   // different to the rest of the tag
   position: absolute;
   display: inline-block;
-  width: $kz-spacing-xs;
-  height: $kz-spacing-xs;
+  width: 8px;
+  height: 8px;
```

Comment:
For optical adjustments like this, I would usually do something like `$kz-spacing-xs + 2px; // optical adjustment`

Reply:
Good call - fixed.